woocommerce/woocommerce-gutenberg-products-block
# Reorganize dependencies in package.json

Fixes #1424.

This PR reorganizes dependencies between `devDependencies` and `dependencies`. It also removes some packages listed in `package.json` which I think were no longer needed.

### How to test the changes in this Pull Request:

1. First run `npm ci` so `node_modules` is cleared and you get the new version of the packages.
2. Do smoke testing of some common tasks: run `npm` commands defined in `package.json` (`npm run lint`, `npm test`, etc.) and verify they don't fail, create a commit and verify prettier and lint still work, etc.
3. Audit the changes and verify the dependencies that were moved belong to the section they are in now.
